### PR TITLE
Make package great again

### PR DESCRIPTION
**Pull request by tspencer244**

### CODE REVIEW

File: src/dateformat.d.ts
Comment (tspencer244) [inline src/dateformat.d.ts:1]: No idea why `@types/dateformat` was giving me grief, had to manually copy the file from that to here for some reason :confused:

File: src/options.ts [hunk 1]
```diff
@@ -97,15 +97,15 @@ export const commandOptions: OptionDefinition[] = [
     alias: 'p',
     description:
       'A filepath expression to look for translation files.\n' +
-      '(e.g. "./resources/locales/\\{\\{lang\\}\\}/\\{\\{namespace\\}\\}.json")\n' +
+      '(e.g. "./resources/locales/\\{\\{lang\\}\\}/\\{\\{namespace\\}\\}on")\n' +
```
Comment (tspencer244) [inline src/options.ts:100]:
```suggestion
      '(e.g. "./resources/locales/\\{\\{lang\\}\\}/\\{\\{namespace\\}\\}.json")\n' +
```

File: .eslintrc.yml [hunk 2]
```diff
@@ -16,6 +16,9 @@ rules:
   eqeqeq: 2
 
 overrides:
+  - files: ['src/dateformat.d.ts']
```
Comment (he-wowers) [inline .eslintrc.yml:19]: Why is dateformat the only file defined in here?

File: src/helpers/writeTranslation.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ export const writeTranslation = (
   options: Options,
 ): void => {
   fetchTranslation(newValue, options.baseLang, lang).then(translated => {
-    // For fun we're going to sanity-check the translations
+    // For fun, we're going to sanity-check the translations
```
Comment (he-wowers) [inline src/helpers/writeTranslation.ts:13]: 😂

**Review by he-wowers — APPROVED** [review]
yeah okay

File: .eslintrc.yml (hunk 2 above)
Comment (tspencer244) [reply .eslintrc.yml:19]: Type definition for a vendor package that uses default export, has to be ignored

File: src/helpers/writeTranslation.ts (hunk 3 above)
Comment (tspencer244) [reply src/helpers/writeTranslation.ts:13]: Correct grammar is best grammar

**Pull request closed without merging**